internal/provider/sap-converged-cloud.go

```diff
@@ -43,3 +43,11 @@ func (p *SapConvergedCloudInputProvider) Provide() Values {
 		FailureTolerance:     &p.FailureTolerance,
 	}
 }
+
+// TODO: Calculate the region parameter using default SapConvergedCloud region. This is to be removed when region is mandatory (Jan 2024).
```

Comment:
Now region is mandatory, so it could be deleted.

Reply:
https://github.com/kyma-project/kyma-environment-broker/pull/1796/commits/b9b79f0e18261cfa78e5ad73d962e24afa058a72